ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/KsqlResourceTest.java

```diff
@@ -507,9 +524,6 @@ public void shouldReturn5xxOnStatementSystemError() {
     givenMockEngine(mockEngine -> {
       EasyMock.expect(mockEngine.parseStatements(EasyMock.anyString()))
           .andReturn(realEngine.parseStatements(ksqlString));
-
-      EasyMock.expect(mockEngine.getQueryExecutionPlan(EasyMock.anyObject(), EasyMock.anyObject()))
```

Comment:
By removing this line you've stopped the test testing what its supposed to - which is what the thing does if `getQueryExecutionPlan` throws and exception.